torch/_refs/nn/functional/__init__.py

```diff
@@ -45,3 +80,78 @@ def elu(
         rhs = refs.expm1(a)
 
     return refs.where(refs.gt(a, 0), a, rhs)
+
+@elementwise_type_promotion_wrapper(
+    type_promoting_args=("a",),
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.OP_MATH,
+)
+def selu(
+    a: TensorLikeType, inplace: bool = False
+) -> TensorLikeType:
+    """
+    Reference implementation of torch.nn.functional.selu
+    """
+
+    if inplace:
+        raise NotImplementedError
+
+    alpha = 1.6732632423543772848170429916717
+    scale = 1.0507009873554804934193349852946
+
+    python_type = utils.dtype_to_type(a.dtype)
+    if not utils.is_weakly_lesser_type(type(alpha), python_type):
+        msg = (
+            "alpha constant of type {0} cannot be safely cast to type {1}!".format(
+                type(alpha), python_type
+            )
+        )
+        raise ValueError(msg)
+    if not utils.is_weakly_lesser_type(type(scale), python_type):
+        msg = (
+            "scale constant of type {0} cannot be safely cast to type {1}!".format(
+                type(scale), python_type
+            )
+        )
+        raise ValueError(msg)
+
+    rhs = refs.mul(alpha, refs.expm1(a))
+
+    # FIXME: If I put the mul on the outside, I hide a stride consistency error!
+    # return refs.mul(scale, refs.where(refs.gt(a, 0), a, rhs))
```

Comment:
Is this fixed after https://github.com/pytorch/pytorch/pull/77542?

Reply:
I am going to merge with #77542 and find out.  I was holding off from removing the `WIP` badge until I made the merge.